static/src/javascripts/projects/commercial/modules/mobile-sticky.js

```diff
@@ -19,10 +19,11 @@ const isInNA = (): boolean =>
 
 export const init = (): Promise<void> => {
     if (
-        config.get('switches.mobileStickyLeaderboard') &&
+        window.location.hash.indexOf('#mobile-sticky') !== -1 ||
```

Comment:
What do you think about moving this if logic into a canRun() function to improve code readability?

Reply:
there's no AB test for the mobile sticky, it only exists for the prebid